app/helpers/mailing_list/unsubscribe_url.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+
+#  Copyright (c) 2012-2017, Jungwacht Blauring Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class MailingList::UnsubscribeUrl
+  include ActionView::Helpers::UrlHelper
+
+  def unsubscribe_link(mailing_list, is_html)
+    if mailing_list.subscribable?
+      unsubscribe_url = unsubscribe_link_string(mailing_list)
+      if is_html
+        link_to('Abmelden / Unsubscribe', unsubscribe_url)
+      else
+        'Abmelden / Unsubscribe: ' + unsubscribe_url
+      end
+    end
+  end
+
+  def unsubscribe_link_string(mailing_list)
```

Comment:
is I mentioned :wink:  the rails path/url helper do what you do manually here!
include Rails.application.routes.url_helpers
group_mailing_list_path, group_mailing_list_url

https://guides.rubyonrails.org/routing.html#path-and-url-helpers